R/dictionary.R

```diff
@@ -192,6 +193,27 @@ vec_unique_count <- function(x) {
   .Call(vctrs_n_distinct, x)
 }
 
+# First location ----------------------------------------------------------
+
+#' Locate first occurrences
+#'
+#' `vec_first_loc()` returns an integer vector giving the location of the
+#' first occurrence of the current element in `x`.
```

Comment:
I find "current" a bit confusing. Perhaps:

> returns an integer vector givin for each element of `x` the location of its first occurence.

Reply:
It would help to compare this function to `vec_duplicate_loc()` and `vec_unique_loc()`.